starter-code/package.json

```diff
@@ -8,13 +8,19 @@
   "dependencies": {
     "bcrypt": "^1.0.2",
```

Comment:
Notice that the dependency "bcryptjs" isn't here, so when I tried to run npm install and run your app, it crashed, since it couldn't find bcryptjs. Once I installed it manually it worked. The problem is probably that you install it without --save and are using an older version of npm. Either update your npm version, or make sure you run 
npm install --save [package_name]